go/vt/discovery/healthcheck.go

```diff
@@ -442,24 +442,27 @@ func (hc *HealthCheckImpl) updateHealth(th *TabletHealth, shr *query.StreamHealt
 		}
 	}
 	if !trivialNonMasterUpdate {
-		if shr.Target.TabletType != topodata.TabletType_MASTER {
+		// we re-sort the healthy tablet list whenever we get a health update for tablets we can route to
+		// tablets from other cells for non-master targets should not trigger a re-sort
+		// they should also be excluded from healthy list
```

Comment:
Need to fix punctuation (and caps). Took me a while to realize that each line is a separate sentence.

Reply:
Apologies. I have fixed it now.